www/app/App.js

```diff
@@ -1,36 +1,41 @@
 import React from 'react';
-import BaseApp from 'next/app';
+import NextApp from 'next/app';
 import Head from 'next/head';
 import keyboardOnlyOutlines from 'keyboard-only-outlines';
 import ReactGA from 'react-ga';
 import getConfig from 'next/config';
-
-import seoImage from '../shared/media/images/seo.png';
 import favicon from '../shared/media/favicons/favicon.ico';
+import SEO_DATA from './App.data';
```

Comment:
I think it's better to have `./App.data.js` since it might seem the file extension is `.data` which is not true. Up for debate, I guess..